CONTRIBUTING.md

```diff
@@ -31,7 +32,11 @@ components, please see also [the sample file](config/samples/kubesphere.yaml).
 ```shell
 make install
 ```
-
+- Copy config/samples/kubesphere.yaml to dir /etc/kubesphere/ .
```

Comment:
I prefer to copy the config file in the current directory for development purpose.